cloudevents/extensions/dataref.md

```diff
@@ -10,7 +10,13 @@ MUST be identical.
 Both `data` and the `dataref` attribute MAY exist at the same time. A middleware
 MAY drop `data` when the `dataref` attribute exists, it MAY add
 the `dataref` attribute and drop the `data` attribute, or it MAY add the `data`
-attribute by using the `dataref` attribute.
+attribute by using the `dataref` attribute. Note that since CloudEvent does
```

Comment:
```suggestion
attribute by using the `dataref` attribute. Note that since CloudEvents does
```

or

```suggestion
attribute by using the `dataref` attribute. Note that since the CloudEvents specification does
```

Reply:
done!